Assets/Scripts/Game/PlayerActivate.cs

```diff
@@ -386,9 +386,16 @@ void Update()
 
                         DaggerfallMessageBox mb;
 
+                        PlayerGPS.DiscoveredBuilding buildingData;
+                        GameManager.Instance.PlayerGPS.GetDiscoveredBuilding(building.buildingKey, out buildingData);
+
                         if (buildingUnlocked &&
                             buildingType >= DFLocation.BuildingTypes.House1 &&
                             buildingType <= DFLocation.BuildingTypes.House4 &&
+                            (buildingData.factionID != (int)FactionFile.FactionIDs.The_Thieves_Guild ||
+                             !GameManager.Instance.GuildManager.HasJoined(FactionFile.GuildGroups.GeneralPopulace)) &&
```

Comment:
Checking whether player has joined TG DB is not needed here, as if they are member, then buildingUnlocked will equal false. Could you simplify you additions to the conditional here please:
`buildingData.factionID != (int)FactionFile.FactionIDs.The_Thieves_Guild &&
buildingData.factionID != (int)FactionFile.FactionIDs.The_Dark_Brotherhood &&`

Reply:
Thanks for the tip, updated.